vars/helmInstall.groovy

```diff
@@ -111,6 +111,13 @@ def call(DockerImage dockerImage, Map params) {
       "--namespace ${namespace}"
     ]
 
+    if (this.env.DISABLE_TRAEFIK_TLS == "true") {
+      //Add global disableTraefikTls flag as true if DISABLE_TRAEFIK_TLS is set to true in jenkins
+        options.add("--set global.disableTraefikTls=true")
+    } else {
+        options.add("--set global.disableTraefikTls=false")
+    }
+
```

Comment:
this is not correct, why would you set an env value that would break all installs?
```suggestion
```